webpack.config.js

```diff
@@ -92,10 +92,10 @@ module.exports = {
     module: {
         rules: [
             {
-                test: /\.js$/,
+                test: /(\.js)|(\.mjs)$/,
```

Comment:
I know you copied this over from DCR, but it might be we could set:

```
experiments: {
    mjs: true,
```

for this?

Also, @nicl is the reason we have to transpile it again, because the module is published with too-modern javascript for the platform?

This actually makes sense, it's a wider question on how we publish *guardian* modules, but it's the first time I've seen an external module having to be resolved this way!

Reply:
Maybe this is a webpack 5 thing, because it didn't seem to be valid config when I tried.